vue/src/views/SmartFlow.vue

```diff
@@ -0,0 +1,183 @@
+<template>
+  <v-container>
+    <v-card>
+      <v-card-title>
+        SmartFlow
+      </v-card-title>
+      <v-card-text>
+        <v-data-table
+          v-model:items-per-page="itemsPerPage"
+          :headers="headers"
+          :items="dagRuns"
+          :items-length="totalItems"
+          :loading="loading"
+          :search="dagSearch"
+          item-name="dag_run_id"
+          class="elevation-1"
+        >
+          <template #top>
+            <v-autocomplete
+              v-model="dag"
+              :items="dagResults"
+              item-text="dag_id"
+              item-value="dag_id"
+              label="Search DAG by ID"
+              hide-details
+              no-filter
+              solo
+              clearable
+              @update:search="fetchDags"
+            />
+          </template>
+          <!-- <template #item.actions="{ item }">
+            <v-icon color="primary" @click="startRun(item)">
+              mdi-play
+            </v-icon>
+            <v-icon color="error" @click="stopRun(item)">
+              mdi-stop
+            </v-icon>
+          </template> -->
+          <template #[`item.state`]="{ item }">
+            <v-chip
+              :color="stateColor(item.state)"
+              dark
+            >
+              {{ item.state }}
+              <v-progress-circular
+                v-if="item.state === 'running'"
+                indeterminate
+                :size="20"
+                class="ml-2"
+              />
+            </v-chip>
+          </template>
+        </v-data-table>
+        <v-btn
+          color="primary"
+          @click="showNewDagRunDialog"
+        >
+          Start New DAG Run
+        </v-btn>
+        <v-dialog
+          v-model="newDagRunDialog"
+          max-width="500"
+        >
+          <v-card>
+            <v-card-title>Enter Region</v-card-title>
+            <v-card-text>
+              <v-text-field
+                v-model="newDagRunTitle"
+                label="DAG Run Title"
+              />
+            </v-card-text>
+            <v-card-text>
+              <v-textarea
+                v-model="newDagRunConf"
+                label="conf"
+              />
+            </v-card-text>
+            <v-card-actions>
+              <v-btn
+                color="primary"
+                @click="startNewRun"
+              >
+                Start
+              </v-btn>
+              <v-btn
+                color="error"
+                @click="newDagRunDialog = false"
+              >
+                Cancel
+              </v-btn>
+            </v-card-actions>
+          </v-card>
+        </v-dialog>
+      </v-card-text>
+    </v-card>
+  </v-container>
+</template>
+
+<script setup lang="ts">
```

Comment:
JS Script at top for consistency based off Vue documentation.  I know I've seen it done this way before but Vue docs actually reccomend script at the top.